<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupForm</class>
 <widget class="QWidget" name="SignupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToDoList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { 
	background-color : rgb(72, 72, 152);
	color: white;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(72, 102, 152, 239), stop:1 rgba(255, 150, 255, 255));
   border-radius: 30px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="logInPushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color : rgb(72, 72, 152);
    color: white; 
    padding: 10px 20px;
    border: 2px solid rgb(72, 72, 50); 
    border-radius: 15px;
	font-family: Arial; 
    font-size: 16px;
    text-align: center; 
	font-weight: bold;
	
}

QPushButton:hover {
    background-color: rgb(45, 45, 95); 
    border-color: rgb(45, 45, 95);
}

</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    background-color : rgb(72, 72, 152);
    color: white; 
    padding: 10px 20px;
    border: 2px solid rgb(72, 72, 50); 
    border-radius: 10px;
	font-family: Arial; 
    font-size: 16px;
	font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    background-color : rgb(72, 72, 152);
    color: white; 
    padding: 10px 20px;
    border: 2px solid rgb(72, 72, 50); 
    border-radius: 10px;
	font-family: Arial; 
    font-size: 16px;
	font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordAgainLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
 {
    background-color : rgb(72, 72, 152);
    color: white; 
    padding: 10px 20px;
    border: 2px solid rgb(72, 72, 50); 
    border-radius: 10px;
	font-family: Arial; 
    font-size: 16px;
	font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
